app/database/attempt_store_compute_all_attempts.go

```diff
@@ -42,16 +42,17 @@ func (s *AttemptStore) ComputeAllAttempts() (err error) {
 
 		// Insert missing attempts for chapters having descendants with attempts marked as 'to_be_recomputed'/'to_be_propagated'.
 		// We only create attempts for chapters which are (or have ancestors which are) visible to the group that attempted
-		// to solve the descendant items.
+		// to solve the descendant items. Chapters with explicit entry (items.entry_participant_type IS NOT NULL) are skipped).
```

Comment:
explicit entry is just `items.requires_explicit_entry` :-)

Reply:
fixed